fuel-vm/tests/cgas.rs

```diff
@@ -138,20 +139,20 @@ fn cgas_uses_min_available_gas() {
 
     let program = vec![
         // load amount of tokens
-        Opcode::ADDI(reg_call_a, REG_FP, CallFrame::a_offset() as Immediate12),
-        Opcode::LW(reg_call_a, reg_call_a, 0),
+        op::addi(reg_call_a, REG_FP, CallFrame::a_offset() as Immediate12),
```

Comment:
Doesn't seem like we need these casts anymore?

```suggestion
        op::addi(reg_call_a, REG_FP, CallFrame::a_offset()),
```

Reply:
We do still need those. `CallFrame::a_offset` returns and `usize` which cannot be passed to `op::addi` which expects an `Imm12` (really `u16`) for this.